ui/app/templates/components/transformation-edit.hbs

```diff
@@ -24,34 +24,19 @@
 {{#if (eq mode "show")}}
   <Toolbar>
     <ToolbarActions>
-      {{#if (eq model.keyType "otp")}}
-        <ToolbarSecretLink
-          @secret={{model.id}}
-          @mode="credentials"
-          @data-test-backend-credentials=true
-          @replace=true
-        >
-         Generate Credential
-       </ToolbarSecretLink>
-      {{else}}
-        <ToolbarSecretLink
-          @secret={{model.id}}
-          @mode="sign"
-          @data-test-backend-credentials=true
-          @replace=true
-        >
-          Sign Keys
-        </ToolbarSecretLink>
-      {{/if}}
+      {{!-- TODO: update these actions, show delete grey out if not allowed --}}
       {{#if (or model.canUpdate model.canDelete)}}
         <div class="toolbar-separator" />
       {{/if}}
       {{#if model.canDelete}}
+      {{!-- TODO only allow deletion when not in use by a role --}}
         <ConfirmAction
           @buttonClasses="toolbar-link"
           @onConfirmAction={{action "delete"}}
+          @confirmTitle="Are you sure?"
+          @confirmMessage="This transformation is not in use by a role and can be deleted."
         >
-          Delete role
+          Delete transformation
         </ConfirmAction>
       {{/if}}
       {{#if (or model.canUpdate model.canDelete)}}
```

Comment:
Someone with only delete capabilities can edit the transformation, is that right?

Reply:
Ah, good catch. I was using a pattern from the template earlier, but they to have canUpdate in order to edit, not canDelete.  I'll amend.